### ISSUE TITLE
error: Found argument '--all' - Documentation update

### ISSUE THREAD

**Issue by pavelbinar** [nushell/nushell.github.io#32]
Tried to install the latest `0.9.0 Release` according to the documentation https://www.nushell.sh/installation.html but got an error:

```
❯ cargo install nu --all --features=stable
error: Found argument '--all' which wasn't expected, or isn't valid in this context
	Did you mean --all-features?

USAGE:
    cargo install --all-features

For more information try --help
```

Command `cargo install nu --all-features --features=stable` works fine.

Please update the documentation page.

**Comment by waldyrious** [comment]
There's a PR already in progress for this: #33.

**Comment by waldyrious** [comment]
@jonathandturner can this be closed? #33 has been merged. :)

**Comment by jonathandturner** [comment]
@waldyrious - great! I'll go ahead and close
